.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: objective-c
-osx_image: xcode7
+osx_image: xcode8.2
 before_install:
 - gem install cocoapods xcpretty --no-rdoc --no-ri --no-document
+- pod repo update
 script:
-- set -o pipefail && xcodebuild test -workspace LockGoogle.xcworkspace -scheme LockGoogle -sdk iphonesimulator ONLY_ACTIVE_ARCH=NO | xcpretty -c
+- set -o pipefail && xcodebuild test -workspace LockGoogle.xcworkspace -scheme LockGoogle -destination 'id=AD8E8C76-910E-437D-88C2-4D6F6EBE3355' | xcpretty -c
```

Comment:
I went through many iterations as the names setup just wouldn't work (see travis PR history). There is an ongoing issue and the 'solution' for now is to specify id.

https://discuss.circleci.com/t/multiple-ios-simulators-available-for-single-os-device-configurations/7854